lib/ae_mdw_web/controllers/tx_controller.ex

```diff
@@ -165,6 +168,48 @@ defmodule AeMdwWeb.TxController do
     json(conn, maybe_add_spend_tx_details(Format.to_map(model_tx)))
   end
 
+  defp extract_query(query_params) do
+    query_params
+    |> Enum.reject(fn {key, _val} -> key in @pagination_param_keys end)
+    |> Enum.reduce_while({:ok, %{}}, fn {key, val}, {:ok, top_level} ->
```

Comment:
no big deal but the tagging of the result could be handled after the reduce_while

Reply:
I agree. I just moved this code from the plug to the controller, because it was tx-specific